src/components/Wallet/NeutronEVMWalletBridges.tsx

```diff
@@ -0,0 +1,61 @@
+import { useWeb3Modal } from '@web3modal/wagmi/react'
+import { useAccount } from 'wagmi'
+import { useCallback } from 'react'
+
+import Button from 'components/common/Button'
+import FullOverlayContent from 'components/common/FullOverlayContent'
+import { useShuttle } from '@delphi-labs/shuttle-react'
+import WalletSelect from 'components/Wallet/WalletSelect'
+import useCurrentWallet from 'hooks/wallet/useCurrentWallet'
+import useStore from 'store'
+
+export default function NeutronEVMWalletBridges() {
```

Comment:
Try to keep chain names out of component names

```suggestion
export default function EvmSupportedBridges() {
```